pkg/apis/flowcontrol/v1alpha1/defaults.go

```diff
@@ -40,6 +40,17 @@ func SetDefaults_FlowSchemaSpec(spec *v1alpha1.FlowSchemaSpec) {
 	}
 }
 
+func SetDefaults_ExemptPriorityLevelConfiguration(eplc *v1alpha1.ExemptPriorityLevelConfiguration) {
+	if *eplc.NominalConcurrencyShares == 0 {
```

Comment:
```suggestion
	if eplc.NominalConcurrencyShares == nil {
```

we only want to default on nil, right?

Reply:
done.